### PR TITLE
group sharing acceptance tests

### PR DESCRIPTION
**Pull request by HariBhandari07**
## Description

Adds acceptance tests for group sharing.

## How Has This Been Tested?

- test environment:
CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/webUISharingInternalGroups/shareWithGroupsEdgeCases.feature [hunk 1]
```diff
@@ -32,3 +32,31 @@ Feature: Sharing files and folders with internal groups
       | group     |
       | ?\?@#%@,; |
       | नेपाली    |
+
+  Scenario: Share file with a user and a group with same name
```
Comment (dpakach) [inline tests/acceptance/features/webUISharingInternalGroups/shareWithGroupsEdgeCases.feature]: I think we can add more scenarios like this. Similar tests were recently added in core. Please check them for reference.
https://github.com/owncloud/core/pull/36283

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6310/

<p>





















































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Trashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6310/

<p>






































</p></details>

File: tests/acceptance/helpers/webdavHelper.js [hunk 2]
```diff
@@ -217,7 +217,7 @@ exports.getSkeletonFile = function (filename) {
     })
 }
 
-exports.uploadFileWithcontent = function (user, content, filename) {
+exports.uploadFileWithContent = function (user, content, filename) {
```
Comment (dpakach) [inline tests/acceptance/helpers/webdavHelper.js]: The function name is changed but it is not updated anywhere. Where is this function called from ?

**Pull request merged**